src/earn/EarnEntrypoint.tsx

```diff
@@ -16,8 +23,33 @@ import { Spacing } from 'src/styles/styles'
 
 export default function EarnEntrypoint() {
   const { t } = useTranslation()
-  const showUKCompliantVariant = getFeatureGate(StatsigFeatureGates.SHOW_UK_COMPLIANT_VARIANT)
 
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+
+  const pools = useSelector(earnPositionsSelector)
+  const poolsSupplied = useMemo(
+    () => pools.filter((pool) => new BigNumber(pool.balance).gt(0)).length,
```

Comment:
nit: since you're just always doing `poolsSupplied > 0` and not use the poolsSupplied count, you can optimize this to directly compute `hasSuppliedPools` using `Array.some` over `filter` + `length`